cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -152,14 +154,18 @@ def __init__(self, scope: core.Construct, construct_id: str, imagebuild: ImageBu
         )
 
         components.append(
-            imagebuilder.CfnImageRecipe.ComponentConfigurationProperty(component_arn=core.Fn.ref("ParallelClusterTag"))
+            imagebuilder.CfnImageRecipe.ComponentConfigurationProperty(
+                component_arn=core.Fn.ref("ParallelClusterTagComponent")
+            )
         )
 
         # ImageRecipe
         imagebuilder.CfnImageRecipe(
             self,
-            id="PClusterImageRecipe",
-            name="-".join(["PCluster", utils.get_installed_version().replace(".", "-"), resources_prefix]),
+            id="ParallelClusterImageRecipe",
+            name="-".join(
```

Comment:
let's remove the version from the name, since it is already in the `version` attribute

Reply:
done